samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,392 @@
+
+* Bluetooth Low Energy and the nRF Device Manager app
+* UART and MCUmgr
+
+The sample demonstrates how to perform a DFU of the Application and Radio firmware.
+
+SUIT is the only DFU procedure supported for the nRF54H20 SoCs.
+To read more about the SUIT procedure, see :ref:`ug_nrf54h20_suit_intro`.
+
+|NCS|-specific SUIT limitations
+===============================
+
+The sample and the SUIT DFU procedure currently only support the Application Core and the Radio Core firmware updates.
+Additionally, this sample is an early version and is only meant to showcase the start of Nordic Semiconductor’s implementation of the SUIT DFU procedure.
+
+User interface
+**************
+
+LED 1:
+    The number of blinks indicates the application SUIT envelope sequence number.
+    The default is set to ``1`` to blink once, indicating "Version 1".
+
+Configuration
+*************
+
+|config|
+
+The default configuration uses UART with Version 1.
+To change the version of the application, configure the :ref:`CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM` Kconfig option.
+This also changes the number of blinks on **LED 1** and sets the :ref:`sequence number <ug_suit_dfu_suit_manifest_elements>` of the SUIT :ref:`envelope <ug_suit_dfu_suit_concepts>`’s manifest.
+
+To use this configuration, include the ``SB_CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=x`` option with the build command, where x is the version number.
+
+For example:
+
+.. code-block:: console
+
+   west build -p -b nrf54h20dk/nrf54h20/cpuapp --sysbuild -- -DCONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=2
+
+If you do not specify this configuration, the sample is built with Version 1.
+
+Configuration options
+=====================
+
+Check and configure the following configuration option for the sample:
+
+.. _CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM:
+
+CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM - Configuration for the sequence number.
+   The sample configuration updates the sequence number of the SUIT envelope, which is reflected as the version of the application in the nRF Device Manager app.
+   The default value is ``1``.
+
+Modify partition sizes
+======================
+
+You can also modify the size and location of the partitions.
+This is done by modifying the values for the desired location and size of the partition in the devicetree :file:`.overlay` files.
+
+* To modify the Application Core’s partition size,  modify the values for ``cpuapp_slot0_partition`` defined in the :file:`zephyr/blob/main/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi` file within the Zephyr repository.
+
+* To modify the DFU partition, modify the values for ``dfu_partition`` defined in :file:`samples/suit/smp_transfer/boards/nrf54h20dk_nrf54h20_cpuapp.overlay`.
+
+    * ``dfu_partition`` is where the update candidate is stored before the update process begins.
+
+Manifest template files
+=======================
```

Comment:
The title suggests to me I would find here a list of the template files. Not finding the detailed list and having the note here that mentions the radio domain manifest template makes me really confused. Only later, when I read about building, can I really understand what this note means. (I guess you mean `radio.suit` here?) For this reason, I think the order of presenting information here is not optimal.

Reply:
That is a good point - I will take a look and try to reorder.
I have also just realised that this note may not be true anymore, and then can be removed (will check with SMEs). But will still try to adjust the order